packages/core/src/git.ts

```diff
@@ -505,10 +509,11 @@ export default class Git {
   }
 
   /** Run a graphql query on the GitHub project */
+  @memoize()
   async graphql<T>(query: string) {
     this.logger.verbose.info("Querying Github using GraphQL:\n", query);
 
-    const data = await graphql<T>(query, {
+    const data = await this.github.graphql<T>(query, {
```

Comment:
Switching to `octokit`'s graphql function will make sure that it is rate/abuse limited properly